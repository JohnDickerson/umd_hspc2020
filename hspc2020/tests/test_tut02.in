8

12345
99999
112233
0
1
12345432
91919000
98769876
