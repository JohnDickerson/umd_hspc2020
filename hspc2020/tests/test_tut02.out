52341
99999
312231
0
1
52341432
99911000
99768876
